about-us.html

```diff
@@ -3,48 +3,62 @@
 <head>
   <meta charset="utf-8">
   <title>Straight to the Point Archery</title>
-  <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+  <link rel="stylesheet" type="text/css" href="styles/lesson-7-css.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital@1&display=swap" rel="stylesheet">
```

Comment:
Remember here, the order in which styles are loaded matters. In this case, you are loading you CSS styles before the fonts are loaded by google. Rule of thumb is generally that you would import a style resetter file like normalize.css first, then any resources like fonts or FontAwesome, and finally your own style at the very last, so that you don't have any unexpected issues or overrides.